README.md

```diff
@@ -96,3 +96,42 @@ You may override our default debug option by providing your own `debug` key.
   }]]
 }
 ```
+
+## React PropTypes removal
+
+This preset will remove propTypes using [babel-plugin-transform-react-remove-prop-types](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types) with the following default options:
+
+```js
+{
+  mode: 'wrap',
+  additionalLibraries: ['airbnb-prop-types'],
+  ignoreFilenames: ['node_modules'],
+}
+```
+
+To disable this transformation, use the `removePropTypes` option:
```

Comment:
by making this default to `true`, it's a major change. i'd prefer to ship it with default `false` first, as a minor, and then switch to changing the default later.

Reply:
ok!

Reply:
@ljharb I added a commit to make this disabled by default.